site/src/api/queries/notifications.ts

```diff
@@ -106,23 +119,22 @@ export const updateNotificationTemplateMethod = (
 		mutationFn: (req: UpdateNotificationTemplateMethod) =>
 			API.updateNotificationTemplateMethod(templateId, req),
 		onMutate: (data) => {
-			const prevData = queryClient.getQueryData<NotificationTemplate[]>(
+			const keys = [
 				systemNotificationTemplatesKey,
-			);
-			if (!prevData) {
-				return;
+				customNotificationTemplatesKey,
+			];
+
+			for (const key of keys) {
+				const prev = queryClient.getQueryData<NotificationTemplate[]>(key);
+				if (!prev) continue;
```

Comment:
```suggestion
				if (!prev) {
					continue;
				}
```

we avoid single line `if`s in the frontend, although unfortunately don't have a lint rule for it rn

Reply:
Addressed in https://github.com/coder/coder/pull/19938/commits/cf7a40363cdee21965aecd373af62a2d94c95916